style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 50px;
+  font-family: "Nata Sans", sans-serif;
+}
```

Comment:
why did you put it on the whole html? i would have put it on the main `div` inside the `body`